packages/gamut/src/Form/Checkbox.tsx

```diff
@@ -23,7 +25,11 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
         ref={ref}
       />
       <label className={s.checkboxLabel} htmlFor={htmlFor}>
-        <div className={s.checkbox}>
+        <div
+          className={cx(s.checkbox, {
+            [s.checkboxMultiline]: multiline,
+          })}
```

Comment:
Nit and very non-blocking: I don't know how this makes you feel, but you could save a few finger taps:

```suggestion
          className={cx(s.checkbox, multiline && s.checkboxMultiline)}
```

Reply:
@JoshuaKGoldberg Pushed up the changes.